UI/UserRoles.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
```

Comment:
can we use enum instead of class?

Reply:
Vlad did UserRole.cs, nobody didn't use this class. And I think, we won't need in this class, because we use AspNetRoles.